source/precalculus/source/05-EL/05.ptx

```diff
@@ -14,14 +14,98 @@
 <introduction>
   In this section, we will explore the properties of logarithms and learn how to manipulate them that will be helpful when we are ready to solve logarithmic equations.
   </introduction>
+
+  <remark>
+      Recall from <xref ref="EL3"/> that we can convert between exponential and logarithmic forms. <me>\log_bx=y</me> is equivalent to <me>b^y=x</me>
+    </remark>
+
+    <activity xml:id="intro-to-one-to-one-property">
+    <introduction>
+      <p>
+        Suppose you are given two equations
+        <m>\log_bM=x</m> and <m>\log_bN=x</m>.
+      </p>
+    </introduction>
+    
+    <task>
+      <statement>
+        <p>
+          Rewrite each logarithmic equation into an exponential equation.
+        </p>
+      </statement>
+      <answer>
+        <p>
+          <m>\log_bM=x \iff b^{x}=M </m>
+        </p>
+        <p>
+          <m>\log_bN=x \iff b^{x}=N </m>
+        </p>
+      </answer>
+    </task>
+
+    <task>
+      <statement>
+        <p>
+          Look at the exponential equations you found in part (a). What conclusion can you make about <m>M</m> and <m>N</m>?
+        </p>
+      </statement>
+      <answer>
+        <p>
+          Because <m>b^{x}=M</m> and <m>b^{x}=N</m>, then it follows that <m>M=N</m>.
+        </p>
+      </answer>
+    </task>
+
+    <task>
+      <statement>
+        <p>
+          Given that both <m>\log_bM</m> and <m>\log_bN</m> are both equal to <m>x</m>, what can you say about <m>\log_bM</m> and <m>\log_bN</m>?
+        </p>
+      </statement>
+      <answer>
+        <p>
+          <m>\log_bM=\log_bN</m>
+        </p>
+      </answer>
+    </task>
+
+    <task>
+      <statement>
+        <p>
+          If given <m>\log_bM=\log_bN</m>, what can you say about <m>M</m> and <m>N</m>? (Refer back to the previous parts of this activity.)
+        </p>
+      </statement>
+      <answer>
+        <p>
+          Based on what we have seen in parts (b) and (c), we can conclude that <m>M=N</m>.
+        </p>
+      </answer>
+    </task>
+    </activity>
+
+    <definition xml:id="def-one-to-one-property">
```

Comment:
Do we have a tag for property to use here instead of definition?

Reply:
Nope.  Maybe `<fact>` will work?